wpa_supplicant/wpa_cli_zephyr.c

```diff
@@ -73,16 +74,17 @@ static int _wpa_ctrl_command(struct wpa_ctrl *ctrl, const char *cmd, int print,
 		os_memcpy(resp, buf, len - 1);
 	}
 
-	if (print) {
+	if (print_ok) {
 			buf[len] = '\0';
 			printf("%s", buf);
+			print_ok = true;
```

Comment:
How would this work? As the initial value of `print_ok` is false it would never enter this branch, no?

Reply:
Yes, right ! was missed out.